styles.css

```diff
@@ -135,13 +125,219 @@ h1 {
   font-size: 16px;
   font-weight: 500;
   line-height: 24px;
-  margin-top: 10px;
+  margin: 0 10px;
   text-transform: uppercase;
 }
 
+/* Work section */
+
+.work {
+  display: grid;
+  margin-right: 25px;
+  padding: 25px;
+}
+
+.project {
+  background-color: #fff;
+  border-radius: 20px;
+  box-shadow: 0 3px 4px 0 #344563;
+  display: flex;
+  flex-wrap: wrap;
+  margin: 20px;
+  padding: 25px;
```

Comment:
- [ ] Great job in implementing your work section design :+1: , but you have to work on small changes to stick to the template you choose. The alignment of working section not properly align. Kindly fix the issue :wrench: 

***Template design***
![Screenshot from 2022-12-02 14-10-41](https://user-images.githubusercontent.com/65129228/205289412-8e33a9a9-1817-41a9-92f1-58aecaeeeb83.png)



***Your design***

![Screenshot from 2022-12-02 14-18-47](https://user-images.githubusercontent.com/65129228/205289231-6b5cea28-67f2-4704-add4-aa5590826719.png)